sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorClient.ts

```diff
@@ -1286,7 +1286,7 @@ export class MetricsAdvisorClient {
    * const client = new MetricsAdvisorClient(endpoint,
    *   new MetricsAdvisorKeyCredential(subscriptionKey, apiKey));
    * const dimensionValues = client
-   *   .listDimensionValuesForDetectionConfiguration(detectionConfigId, startTime, endTime, dimensionName);
+   *   .listAnomalyDimensionValues(detectionConfigId, startTime, endTime, dimensionName);
```

Comment:
update the summary comments `   * Returns an async iterable iterator to list dimension values for a detection configuration.` to maybe `   * Returns an async iterable iterator to list dimension values of anomalies detected by a detection configuration.`